### ISSUE TITLE
Pandoc 2.10 and Underlines

### ISSUE THREAD

**Issue by iandol** [htdebeer/paru#58]
Trying to compile a document using Pandoc 2.10 and the new underline type, I am getting an error when running a Paru filter:

pandocomatic.yaml template:
```yaml
templates:
  test:
    pandoc:
      from: markdown
      to: html5
      filter:
        - ./noop.rb
    metadata:
      lang: 'EN-GB'
```

noop.rb filter:
```ruby
#!/usr/bin/env ruby

require 'paru/filter'

Paru::Filter.run do
  stop!
end
```

test.md
```markdown
---
title: "Underline test"
pandocomatic_:
  use-template: test
---

# Abstract #

[Lørem ipsum dolør sit amet]{.underline} , eu ipsum movet vix, veniam låoreet posidonium te eøs, eæm in veri eirmod. Sed illum minimum at, est mægna alienum mentitum ne. Amet equidem sit ex. Ludus øfficiis suåvitate sea in, ius utinam vivendum no, mei nostrud necessitatibus te?  
```

The error I get is the following:

```
➜ pandocomatic -b -c pandocomatic.yaml test.md
pandoc	--from=markdown \
	--to=html5 \
	--filter=noop.rb
Error running filter noop.rb:
Error in $.blocks[1].c[0]: mempty
Error running pandoc => error while running:

pandoc	--from=markdown \
	--to=html5 \
	--filter=noop.rb

Pandoc responded with:

Error running filter noop.rb:
Error in $.blocks[1].c[0]: mempty
```

If I change the class of the [inline]{} to something other than `underline`, then it compiles without issue...

```
➜ pandoc -v
pandoc 2.10
Compiled with pandoc-types 1.21, texmath 0.12.0.2, skylighting 0.8.5
Default user data directory: /Users/ian/.local/share/pandoc or /Users/ian/.pandoc
Copyright (C) 2006-2020 John MacFarlane
Web:  https://pandoc.org
This is free software; see the source for copying conditions.
There is no warranty, not even for merchantability or fitness
for a particular purpose.
➜ pandocomatic -v
Pandocomatic version 0.2.7.2
© 2014—2020 Huub de Beer <Huub@heerdebeer.org>

Pandocomatic is free software; pandocomatic is released under the GPLv3.

For more information about pandocomatic run 'pandocomatic --help' or read its
documentation at https://heerdebeer.org/Software/markdown/pandocomatic/.

➜ gem list | grep paru
paru (0.4.1)
```

**Comment by htdebeer** [comment]
I can reproduce the issue. I will create a test and a fix.

**Comment by htdebeer** [comment]
Fixed in Paru 0.4.1.1.
